src/help/zaphelp/contents/releases/1_2_0.html

```diff
@@ -37,7 +37,7 @@ <H3>XML Reports</H3>
 
 <H3>Manual Request Editor and Resend dialogs</H3>
 Requests submitted by the <a href="../ui/dialogs/man_req.html">Manual Request Editor</a> and
```

Comment:
Just a small improvement. Use the link for both "Manual Request Editor and Resend dialogs"

`<a href="../ui/dialogs/man_req.html">Manual Request Editor and Resend dialogs</a> `

Reply:
Change done.